### PR TITLE
Updated list command to list playbooks.

### PR DESCRIPTION
**Pull request by asa1997**
Fixed issue #30

### CODE REVIEW

**Review by jobyko — APPROVED** [review]
@asa1997 
LGTM

**Comment by asa1997** [comment]
@panickervinod Please merge the pull request.

**Comment by panickervinod** [comment]
@asa1997 @jobyko Please remove hard coding of file name temp.txt in function

File: src/main/bash/commands/besman-list.sh [hunk 1]
```diff
@@ -2,10 +2,18 @@
 
 function __bes_list {
 
+if [[ -n $1 ]]; then
+    ls $BESMAN_DIR/playbook >> $HOME/temp.txt
```
Comment (panickervinod) [inline src/main/bash/commands/besman-list.sh:6]: @asa1997 Please remove hardcoding of tempfile name as temp.txt

**Pull request merged**